groups/prometheus/locals.tf

```diff
@@ -0,0 +1,18 @@
+data "vault_generic_secret" "secrets" {
+  path = "team-${var.team}/${var.account_name}/${var.region}/${var.repository_name}"
```

Comment:
You'll have to add `${var.environment}` to this path like I did [here](https://github.com/companieshouse/concourse-terraform/pull/155/commits/02ba7eeb2328da41262426089ebe46b15552472e). I forgot to include it in the initial cut